pkg/prometheus/promcfg_test.go

```diff
@@ -5474,6 +5474,44 @@ scrape_configs: []
 remote_read:
 - url: http://example.com
   remote_timeout: 30s
+`,
+		},
+		{
+			version: "v2.26.0",
+			remoteRead: monitoringv1.RemoteReadSpec{
+				URL:             "http://example.com",
+				FollowRedirects: pointer.Bool(true),
+			},
+			expected: `global:
+  evaluation_interval: 30s
+  scrape_interval: 30s
+  external_labels:
+    prometheus: default/test
+    prometheus_replica: $(POD_NAME)
+scrape_configs: []
+remote_read:
+- url: http://example.com
+  remote_timeout: 30s
+  follow_redirects: true
+`,
+		},
+		{
+			version: "v2.26.0",
+			remoteRead: monitoringv1.RemoteReadSpec{
+				URL:             "http://example.com",
+				FollowRedirects: pointer.Bool(false),
```

Comment:
Adding the same test but just different booleans might be too much 😅 

They follow the same codebase, so only one already covers everything

Reply:
@ArthurSens agree :slightly_smiling_face: Should we keep it for just one boolean in this case? (e.g. `false`)

Reply:
yeah, just picking one is fine!